official/nlp/transformer/utils/tokenizer.py

```diff
@@ -342,18 +357,26 @@ def _count_tokens(files, file_byte_limit=1e6, correct_strip=True):
     Dictionary mapping tokens to the number of times they appear in the sampled
     lines from the files.
   """
+  if master_char_set is None:
+    master_char_set = _ALPHANUMERIC_CHAR_SET
+
   token_counts = collections.defaultdict(int)
 
   for filepath in files:
     with tf.io.gfile.GFile(filepath, mode="r") as reader:
       file_byte_budget = file_byte_limit
       counter = 0
       lines_to_skip = int(reader.size() / (file_byte_budget * 2))
-      for line in reader:
+      for i,line in enumerate(reader):
+        if i%100==0:
+          tf.compat.v1.logging.debug("Counting tokens, byte budget left for '%s:%d': %d" %
+                                     (filepath, i, file_byte_budget))
+
         if counter < lines_to_skip:
           counter += 1
         else:
           if file_byte_budget < 0:
+            tf.compat.v1.logging.debug("Byte budget of '%s' is over" % (filepath,))
             break
```

Comment:
overall, do we really need this debug logging?

Reply:
Ok, it's really no that important.